src/main/resources/public/raml/resourceTypes/failed-notifications.raml

```diff
@@ -1,11 +1,11 @@
-description: This class searches for missed notifications in the database.
+description: This class searches for failed notifications in the database.
 get:
-  description: This method returns missed notification(s) given a subscription name.
+  description: This method returns failed notification(s) given a subscription name.
   queryParameters:
     subscriptionName:
       displayName: subscriptionName
       type: string
-      description: SubscriptionName of the Missed Notification
+      description: SubscriptionName of the failed notification
```

Comment:
In this case this is a descriptive sentence. Shouldn't it be 'Subscription name of the failed notification'?